### PR TITLE
adding ggF higgs->aa->2mu2tau

### PR DESCRIPTION
**Pull request by anirban1982**
adding card for one mass point of ggF higgs->aa->2mu2tau

@efeyazgan @perrozzi please review them.

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ggh01_M125_Toa01a01_M5_Tomumutautau_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+#          reweighting into account!                                 *
+#*********************************************************************
+   T = use_syst
+#
+#**************************************
+# Parameter of the systematics study
+#  will be used by SysCalc (if installed)
+#**************************************                                  
+#
+0.5 1 2 = sys_scalefact  # factorization/renormalization scale factor
```
Comment (efeyazgan) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ggh01_M125_Toa01a01_M5_Tomumutautau_run_card.dat]: @perrozzi @kdlong @qliphy @agrohsje  what do we do with these lines?
https://github.com/anirban1982/genproductions/blob/20399ef110f1243311178be5b9c4a2b3a0ffb0b1/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ggh01_M125_Toa01a01_M5_Tomumutautau_run_card.dat#L276-L281

Comment (anirban1982) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ggh01_M125_Toa01a01_M5_Tomumutautau_run_card.dat]: @perrozzi @kdlong @qliphy @agrohsje @efeyazgan good point indeed. I saw in

https://twiki.cern.ch/twiki/bin/viewauth/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

says

SysCalc is not used anymore and these lines should be removed: 

Should I remove the line 276 to 281?

https://github.com/anirban1982/genproductions/blob/20399ef110f1243311178be5b9c4a2b3a0ffb0b1/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ggh01_M125_Toa01a01_M5_Tomumutautau_run_card.dat#L276-L281

Comment (kdlong) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ggh01_M125_Toa01a01_M5_Tomumutautau_run_card.dat]: They are irrelevant, so probably best to remove them. Syscalc is used but is configured independently in the gridpack_generation scripts.

**Comment by anirban1982** [comment]
@kdlong thanks, removed lines 276 to 281 in the run card.

@efeyazgan @perrozzi could you please merge it now?

**Pull request merged**